operations/mimir/read-write-deployment/write.libsonnet

```diff
@@ -30,6 +30,11 @@
   mimir_write_zone_b_args:: $.ingester_zone_b_args {},
   mimir_write_zone_c_args:: $.ingester_zone_c_args {},
 
+  mimir_write_node_affinity_matchers:: [],
```

Comment:
This doesn't seem like it's used anywhere?

Reply:
The per-zone matchers (lines below) inherit from it. It's handy when you want to set a node affinity matcher for mimir-write regardless the zone.